package.json

```diff
@@ -7,25 +7,26 @@
   "author": "Tobias Koppers @sokra",
   "homepage": "https://github.com/webpack/webpack-dev-middleware",
   "bugs": "https://github.com/webpack/webpack-dev-middleware/issues",
-  "main": "index.js",
+  "main": "lib/index.js",
   "engines": {
     "node": ">= 6"
   },
   "scripts": {
     "commitlint": "commitlint --from=master",
-    "lint": "eslint --cache lib test",
+    "lint:prettier": "prettier \"{**/*,*}.{js,json,md,yml,css}\" --list-different",
+    "lint:js": "eslint --cache lib test",
+    "lint": "npm-run-all -l -p \"lint:**\"",
     "release": "standard-version",
     "security": "npm audit",
-    "test:only": "jest",
-    "test:coverage": "npm run test:only -- --coverage",
-    "test:watch": "npm run test:only --watch",
+    "test:only": "cross-env NODE_ENV=test jest",
+    "test:watch": "cross-env NODE_ENV=test jest --watch",
+    "test:coverage": "cross-env NODE_ENV=test jest --collectCoverageFrom=\"lib/**/*.js\" --coverage",
```

Comment:
It is better to separate to jest.config.js?

Reply:
I think it is more simple than using `jest.config.js`. Maybe we revisit this in future.

Reply:
ok